frontend/src/app/assess/[projectId]/fairness-bias/api-endpoint/job/[jobId]/page.tsx

```
@@ -125,34 +125,34 @@ export default function FairnessJobPage() {
+          <h2 className="text-xl font-semibold text-foreground mb-2">
             Unable to load this job
           </h2>
-          <p className="text-gray-600 dark:text-gray-400 mb-6">{error}</p>
+          <p className="text-muted-foreground mb-6">{error}</p>
           <div className="flex gap-3 justify-center">
             <button
               onClick={() => fetchStatus(true)}
-              className="px-4 py-2 rounded-lg bg-purple-600 text-white hover:bg-purple-700 transition"
+              className="px-4 py-2 rounded-lg bg-primary text-primary-foreground hover:bg-primary/90 transition"
             >
               Retry
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)